aws-frauddetector-detector/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,39 @@
+{
+  "DetectorId": "cfntestdetector1",
+  "DetectorVersionStatus": "ACTIVE",
+  "RuleExecutionMode": "ALL_MATCHED",
+  "Tags": [
+    {
+      "Key": "cfncontracttest",
+      "Value": "1"
+    }
+  ],
+  "Description": "detector for cfn contract tests",
+  "Rules": [
+    {
+      "RuleId": "cfntestrule",
+      "DetectorId": "cfntestdetector1",
+      "Expression": "1 == 5",
+      "Language": "DETECTORPL",
+      "Outcomes": [
+        {
+          "Inline": false,
+          "Name": "reference_outcome_for_detector",
+          "Arn": "arn:aws:frauddetector:us-west-2:123123123123:outcome/reference_outcome_for_detector"
```

Comment:
Will the use of "arn:aws" work in all partitions?

Reply:
No, this will not work in other partitions. Ideally, we would be using a !Ref or !GetAtt here, but I'm not sure those CFN functions are supported in the contract test input.

Instead of using non-Inline examples, we can keep contract tests using Inline definitions for outcomes + eventtype, that way there are no issues. We should still keep a set of inputs with referenced dependencies to test against manually, but it might be difficult to maintain contract test success with these referenced dependencies.